docs/scanners.md

```diff
@@ -30,7 +30,7 @@ class MyKeyboard(KMKKeyboard):
             row_pins=self.row_pins,
             # optional arguments with defaults:
             columns_to_anodes=DiodeOrientation.COL2ROW,
-            interval=0.02,
+            interval=0.02,  # Debounce time
```

Comment:
time in what unit? seconds? milliseconds? something else?

Reply:
One would assume ms I suppose. Unspecified. "Unit's of time" though for sure based on upstream's documentation https://learn.adafruit.com/key-pad-matrix-scanning-in-circuitpython/advanced-features